src/x/opentelemetry/config.go

```diff
@@ -70,22 +67,9 @@ func (c Configuration) NewTracerProvider(
 		return nil, fmt.Errorf("failed to create resource: %w", err)
 	}
 
-	driverOpts := []otlptracegrpc.Option{
-		otlptracegrpc.WithEndpoint(c.Endpoint),
-		otlptracegrpc.WithDialOption(grpc.WithBlock()),
-	}
-	if c.Insecure {
-		driverOpts = append(driverOpts, otlptracegrpc.WithInsecure())
-	}
-	driver := otlptracegrpc.NewClient(driverOpts...)
-	traceExporter, err := otlptrace.New(ctx, driver)
-	if err != nil {
-		return nil, fmt.Errorf("failed to trace exporter: %w", err)
-	}
-
 	// Register the trace exporter with a TracerProvider, using a batch
 	// span processor to aggregate spans before export.
-	batchSpanProcessor := sdktrace.NewBatchSpanProcessor(traceExporter)
+	batchSpanProcessor := sdktrace.NewBatchSpanProcessor(nil)
```

Comment:
Just so I understand--what's the behavior now if someone invokes this package? Does it crash, just noop or... ? I'm fine with basically whatever, we just need to call it out explicitly.